client/my-sites/themes/theme-upload/index.jsx

```diff
@@ -312,9 +300,11 @@ class Upload extends Component {
 				<div className="theme-upload__description">{ theme.description }</div>
 				<div className="theme-upload__action-buttons">
 					<Button onClick={ this.onTryAndCustomizeClick }>{ tryandcustomize.label }</Button>
-					<Button primary onClick={ this.onActivateClick }>
-						{ activate.label }
-					</Button>
+					{ this.props.activeTheme !== theme.id && ! this.props.isTransferCompleted && (
```

Comment:
This looks wrong? We shouldn't allow the theme to be activated if the transfer has not been completed, but here, we're allowing it only if the transfer is still in progress 🤔 Am I missing anything?

Reply:
isTransferCompleted, in context of theme upload, means that the theme was also installed on the site. The complete action is triggered only after the polling logic determines that the theme is active on the site.